### DISCUSSION TITLE
Looks like the `getExitCode` never returns 0 for the `ConsoleErrorEvent`, so this logic seems that useless

### DISCUSSION THREAD

**Discussion by yankewei** [symfony/symfony#57957] [Q&A] [answered]
[            if (0 === $exitCode = $event->getExitCode()) {](https://github.com/symfony/symfony/blob/031dde708e908ae097dbd747aefab13c64583e81/src/Symfony/Component/Console/Application.php#L1054)

**Comment by mbabker** [comment, marked as answer]
An event listener can call the event's `setExitCode()` method and set it to 0.

**Reply by yankewei** [reply]
Thank you, I missed the `setExitCode` method.
